### PR TITLE
Add support board Meow Meow

### PR DESCRIPTION
**Pull request by sabas1080**
Hi 

Add support Meow Meow board by Electronic Cats

https://github.com/ElectronicCats/MeowMeow

Thanks

### CODE REVIEW

**Comment by sabas1080** [comment]
ready @pelikhan

File: README.md [hunk 1]
```diff
@@ -1,28 +1,61 @@
-# MakeCode Maker
```
Comment (pelikhan) [inline README.md]: Revert changes to this file.

File: docs/examples/blinky.md [hunk 2]
```diff
@@ -12,5 +12,5 @@ forever(function() {
 ```
 
 ```package
-adafruit-metro-m0-express
+electroniccats-meow-meow
```
Comment (pelikhan) [inline docs/examples/blinky.md]: revert

File: docs/examples/glow-light.md [hunk 3]
```diff
@@ -10,5 +10,5 @@ forever(function () {
 ```
 
 ```package
-adafruit-metro-m0-express
+electroniccats-meow-meow
```
Comment (pelikhan) [inline docs/examples/glow-light.md]: revert

File: docs/examples/glowy.md [hunk 4]
```diff
@@ -16,5 +16,5 @@ forever(function () {
 ```
 
 ```package
-adafruit-metro-m0-express
+electroniccats-meow-meow
```
Comment (pelikhan) [inline docs/examples/glowy.md]: revert

**Comment by sabas1080** [comment]
I have solved the notes @pelikhan , but now I have the following error
```
building libs/blocksprj
created native image in offline cache for project libs/blocksprj: /home/travis/build/Microsoft/pxt-maker/built/hexcache/b7f9e64df9fa54c8759ec125d33ed7191d842eb11048924b104cf8537f4d9e65.hex
INTERNAL ERROR: SyntaxError: Unexpected string in JSON at position 335
    at JSON.parse (<anonymous>)
    at Object.readJson (/home/travis/build/Microsoft/pxt-maker/node_modules/pxt-core/built/nodeutil.js:245:17)
    at saveThemeJson (/home/travis/build/Microsoft/pxt-maker/node_modules/pxt-core/built/pxt.js:131782:39)
    at /home/travis/build/Microsoft/pxt-maker/node_modules/pxt-core/built/pxt.js:132078:9
    at tryCatcher (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/home/travis/build/Microsoft/pxt-maker/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

**Comment by pelikhan** [comment]
Make sure to pull both pxt and pxt-common-packages.

**Comment by sabas1080** [comment]
@pelikhan  I have updated pxt and commons and error

```
building libs/blocksprj
native image already in offline cache for project libs/blocksprj: /Users/sabas/Documents/proyectoshd/pxt-produccion/pxt-maker/built/hexcache/bd7acdaa5b64df262a9bb5173a2d6d1521b4ef3fb7eb3b26beb7eccf1440e1e8.hex

***
*** Build failed: target build failed: Unexpected string in JSON at position 335
SyntaxError: Unexpected string in JSON at position 335
    at JSON.parse (<anonymous>)
    at Object.readJson (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/built/nodeutil.js:245:17)
    at saveThemeJson (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/built/pxt.js:131782:39)
    at /Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/built/pxt.js:132078:9
    at tryCatcher (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/sabas/Documents/proyectoshd/pxt-produccion/pxt/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)
    at processImmediate (timers.js:649:5)
***
```

**Comment by sabas1080** [comment]
`npm install` downgrade my dependecies

captura de pantalla 2018-10-04 a la s 18 08 16

**Comment by sabas1080** [comment]
@pelikhan I'm sorry it was my error, I have corrected all changes, thanks

**Comment by sabas1080** [comment]
I have cleaned everything and created a new #69 

Thanks @pelikhan

**Pull request closed without merging**